presto-parser/src/main/java/io/prestosql/sql/tree/CommentTable.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.sql.tree;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public final class CommentTable
```

Comment:
@martint  My understanding is the suggestion means SqlBase.g4 also should be changed. Is it right?

Current:
```g4
    | COMMENT ON TABLE qualifiedName IS string                         #commentTable
```

New:
```g4
    | COMMENT ON commentObject qualifiedName IS string                 #comment
...
commentObject
    : TABLE          #commentTable
    ;
```

Reply:
No, the grammar is fine. Rather than introducing a literal object, we can add a Type enum like other AST modes use. It can be refactored into object later if needed.

Reply:
I see. Thank you for answering.